.gitignore

```diff
@@ -234,3 +234,4 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+.vscode
```

Comment:
Fine in this PR, but just a reminder to myself to PR this change https://github.com/seamapi/fake-template/commit/c69e7da4a9466cbdb579db79138cb30f091bc134